.eslintrc

```diff
@@ -62,6 +62,7 @@ find node_modules/eslint-config-airbnb -name '*.js'|xargs sed -i "s/': 2/': 1/"
     "jsx-a11y/anchor-is-valid": ["warn", { "aspects": ["invalidHref"] }],
     "import/no-extraneous-dependencies": 0,
     "import/no-unresolved": 0,
-    "import/extensions": 0
+    "import/extensions": 0,
+    "global-require": 0
```

Comment:
what does disabling global-require do?

Reply:
it allows to "require" a module inside a function or an expression, not only globally in the top of a source file

Reply:
what in particular requires (no pun intended) requiring a module not at the top of a file?

Reply:
code splitting